src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -2769,8 +2826,18 @@ extern "C" EXPORT_API MonoClass* EXPORT_CC mono_unity_class_get_generic_type_def
 
 extern "C" EXPORT_API gboolean EXPORT_CC mono_unity_class_has_failure (MonoClass * klass)
 {
-    ASSERT_NOT_IMPLEMENTED;
+#ifndef DACCESS_COMPILE
```

Comment:
@joncham I've added a basic impl for mono_unity_class_has_failure  as this is very spammy not implemented call during asset importing